frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -220,6 +222,12 @@ const GraphicalSelector = () => {
     graph?.changeSize(ref.current?.scrollWidth ?? 0, ref.current?.scrollHeight ?? 0);
   }, [graph, sidebar]);
 
+  const items = [
+    { label: 'Course Info', key: 'item-1', children: 'Content 1' }, // remember to pass the key prop
```

Comment:
rename the key to something more meaningful? i.e. `course-info`, `program-structure`, `help`. also remove the comment here